packages/dont-crypto/README.md

```diff
@@ -0,0 +1,3 @@
+# dont-crypto
+
+This is the directory where the dont cryptography implementation will live
```

Comment:
Would you mind adding some setup steps so I can try this out? I'm not sure about the `cabal` stuff or what I'd need to build/deploy haskell serverless.